crates/storage/codecs/derive/src/arbitrary.rs

```diff
@@ -79,6 +79,13 @@ pub fn maybe_generate_tests(args: TokenStream, ast: &DeriveInput) -> TokenStream
                         #(#roundtrips)*
                     });
                 }
+
+                #[test]
+                fn malformed_rlp_checks() {
```

Comment:
I'm not sure if I understand the issue properly. Is this what intended to do?

Reply:
yes, the entire point of the referenced issue is to move the malformed header check out of the proptest test so they don't run multiple times (slows down CI)